solutions_automation/deployer.py

```diff
@@ -54,7 +54,7 @@ def deploy_discourse(solution_name, smtp_email, stmp_host, stmp_password, wg_con
     return DiscourseAutomated(
         solution_name=solution_name,
         smtp_email=smtp_email,
-        stmp_username=stmp_host,
+        stmp_host=stmp_host,
```

Comment:
not smtp?

Reply:
This line should be smtp, but the previous line should be `host_email`
Fixed in 066452f2b057ae73c651b4769bbaf6ffe3c3b8aa